### PR TITLE
New tags

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3426/commits/a632288b05948787a31f975fabb149656d13c8df is at https://pr3426.build.csb.dev/s/new.

File: packages/common/src/design-language/theme.ts [hunk 1]
```diff
@@ -17,8 +17,9 @@ const theme = {
   // 7 - 32
   // 8 - 36
   // 9 - 40
-  space: [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
-  sizes: [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
+  // 10 - 44
```
Comment (siddharthkp) [inline packages/common/src/design-language/theme.ts]: @SaraVieira fyi: the scale in comments is incorrect
0 * 4 = 0
1 * 4 = 4
2 * 4 = 8
3 * 4 = 12
...
10 * 4 = 40
11 * 4 = 44

File: packages/common/src/design-language/typography.ts [hunk 2]
```diff
@@ -1,15 +1,4 @@
...
-  11,
-  13,
-  16,
-  19,
-  24,
-  28,
-  33,
-  40,
-];
+export const fontSizes = [0, 9, 11, 13, 16, 19, 24, 28, 33, 40];
```
Comment (siddharthkp) [inline packages/common/src/design-language/typography.ts]: i knew it! I knew danny would make something tiny :P

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+const InputComponent = ({ className, ...other }) => (
+  <Input
+    {...other}
+    placeholder="Add Tag"
+    css={css({
+      maxWidth: '45px',
+      fontSize: 1,
+      paddingX: 1,
+      height: '19px',
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx]: height: 5? (20px is on the grid)

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    <Tag {...props} onRemove={() => props.onRemove(props.key)} />
+  </Element>
+);
+
+const InputComponent = ({ className, ...other }) => (
+  <Input
+    {...other}
+    placeholder="Add Tag"
+    css={css({
+      maxWidth: '45px',
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx]: Interesting width choice, why 45?

File: packages/common/src/design-language/theme.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/common/src/design-language/theme.ts]: Fixedf and pushged :)

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx (hunk 4 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx]: Design

Comment (siddharthkp) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx]: bad design 😢

Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx]: It was 44 sorry, fixed

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx (hunk 3 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/EditableTags.tsx]: Good point! Changed!

**Pull request merged**